src/static-template.js

```diff
@@ -17,7 +17,8 @@
 import {devAssert} from './log';
 import {map} from './utils/object.js';
 
-let container;
+let htmlContainer;
```

Comment:
/cc @alanorozco, does this makes sense?

Reply:
LGTM.

We need to ensure correct usage by modifying the lint rule created by @jridgewell and
adapt the html minifier transform by @kristoferbaxter so it takes this template tag into account.